extensions/wikia/Email/EmailController.class.php

```diff
@@ -145,13 +173,24 @@ public function main() {
 	/**
 	 * Create an error response for any exception thrown while creating this email
 	 *
-	 * @param ControllerException $e
+	 * @param \Exception $e
 	 *
 	 */
-	protected function setErrorResponse( ControllerException $e ) {
+	protected function setErrorResponse( \Exception $e ) {
```

Comment:
So it turns out that Nirvana change an exception thrown (say one based on our ```Email\ControllerException```) and re-throw it as its own ```WikiaDispatchedException```.  Given that we use the Nirvana request routing a few times during email creation, we need to catch all errors.